libraries/AC_Planck/AC_Planck.h

```diff
@@ -75,6 +75,27 @@ class AC_Planck {
   //Get tag attitude
   Vector3f get_tag_att(){return _tag_est.tag_att_cd;};
 
+  //Tether timed out
+  bool is_tether_timed_out() { return ((AP_HAL::millis() - _tether_status.timestamp_ms) > 5000); };
```

Comment:
I think the timeout should be set to the FS_GCS_TIMEOUT_MS macro.

Reply:
Yeah, good call.  Or maybe a dedicated parameter?

Reply:
Yeah, a param would be best, though we need to make sure to replace all instances of FS_GCS_TIMEOUT_MS with that param. The we could also pass that param to the DECK TetherControl app so they're all synched.

Reply:
Maybe use params for the other timeout thresholds too? Although I don't love the idea of adding so many params, perhaps it's necessary to be throrough.

Reply:
Further investigation shows this isn't trivial; `FS_GCS_TIMEOUT_MS` is part of the ArduCopter/Copter scope and classes, and isn't easily exposed to `AC_Planck`. Since `FS_GCS_TIMEOUT_MS` is hard-coded (and not a parameter), I'm inclined to leave these as hardcoded values.

Reply:
Fair enough